modeUtil.py

```diff
@@ -5,3 +5,10 @@ def checkOrderById(arrangeMode):
             return True
         else:
             return False
+
+    @staticmethod
+    def checkConditionAssigned(arrangeMode):
+        if arrangeMode in [3,4,5,7]:
```

Comment:
But...if `arrangeMode` is `7`, then this function would return 'Ture`. Doesn't it matter?